qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy

```diff
@@ -89,7 +89,7 @@ class DeploymentEventGraphQLTest extends BaseSpecification {
 
     private final gqlService = new GraphQLService()
 
-    @Category(GraphQL)
+    @Category(BAT)
```

Comment:
Changed this to make sure the tests were executed for the PR.  Will change that back to GraphQL after review.

Reply:
I think this test should be BAT https://github.com/stackrox/stackrox/pull/3340

Reply:
Fine with me.